src/SolutionTemplate/UnoSolutionTemplate.VISX/UnoSolutionTemplate.VISX.csproj

```diff
@@ -83,9 +83,7 @@
     <None Include="source.extension.vsixmanifest">
       <SubType>Designer</SubType>
     </None>
-    <None Include="winjs-symbols.ttf">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

Comment:
same comment here, I would preserve the `CopyToOutputDirectory` attribute to be on the safe side